lib/python/rose/apps/rose_arch.py

```diff
@@ -39,6 +39,13 @@
 from time import gmtime, strftime, time
 
 
+class RoseArchDuplicateError(ConfigValueError):
+
+    """An exception raised if dupluicate archive targets are provided."""
```

Comment:
Typo: 'dupl~~u~~icate'

Reply:
Merging and amending typo quickly with a separate PR, as advised by Matt.

Reply:
Thanks!